src/camvidlog2/cli.py

```
@@ -52,63 +58,67 @@ def load(
     roll: Annotated[int, typer.Option("--rolling", "-r", min=0)] = 0,
+    json: Annotated[Path | None, typer.Option()] = None,
 ):
-    df = data_load(db)
-    if df is None:
+    video_embeddings = data_load(db)
+    if video_embeddings is None:
         raise ValueError("Unable to load database")
 
+    if json:
+        if queries:
+            raise ValueError("Cannot use --json and [QUERIES]")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)